src/App.js

```diff
@@ -0,0 +1,19 @@
+import { HomeScreen } from "./components/HomeScreen";
+import { LoginScreen } from "./components/Login/LoginScreen";
+import "./styles.css"
+import { Route, Routes } from "react-router";
+import { Page } from "./components/Page";
+import { VideoPlayer } from "./components/VideoPlayer";
+
```

Comment:
u can group them in an index file ....
then will look like this `import {Page, VideoPLayer} from "./components"`

Reply:
I'll do this! thanks! :D